### PR TITLE
Added license URI to orignal author

### PR DESCRIPTION
**Pull request by VincentTam**
## Description

* Added proper attribution to the original author in the README, LICENSE, and the site footer so that on every page generated by this theme, there'll be proper attribution to the [original author](https://aj.lkn.io/).
* Changed the text width of README to 80.
* Removed the carriage return `\r` in the :br: Brazillian Portuguese locale.
* Added `{{ .Site.Hugo.version }}` for displaying the version number of Hugo during site compilation.  This helps us understand which Hugo version the user is using in case (s)he opens an issue here.
* Fixed some typo in README.
    - I prefered splitting "HTML5 UP" into two words instead of "HTML5UP".  That's what can be seen on the original author's website: https://aj.lkn.io/
    - I've put "CSS Grid" instead of "CSS-Gridd" based on [DuckDuckGo's search results](https://duckduckgo.com/?q=css+grid&ia=web).

## Motivation and Context

Resolve @balibebas's follow-up comment in #47, and introduce a link "Hugo v0.xx.x" displaying the actual version of Hugo used by the site owner.  This might facilitate our debugging process.

## Screenshots (if appropriate):

Screenshot_2019-08-12 Hugo Future Imperfect Slim in fr

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the [Contributing Document](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/.github/CONTRIBUTING.md).
- [x] My code follows the [code style](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/.github/CONTRIBUTING.md#Style-Guide) of this project.
- [ ] My change requires a change to the [documentation](https://github.com/pacollins/hugo-future-imperfect-slim/wiki).
- [x] I have updated the documentation, including `theme.toml`, accordingly.

## Reference

1. [Full text of CC BY 3.0 License](https://tldrlegal.com/license/creative-commons-attribution-(cc)#fulltext)

    > \4. Restrictions. The license granted in Section 3 above is expressly made subject to and limited by the following restrictions:
    >
    >  (i) **the name of the Original Author (or pseudonym, if applicable)** if supplied, and/or if the Original Author and/or Licensor designate another party or parties (e.g., a sponsor institute, publishing entity, journal) for attribution ("Attribution Parties") in Licensor's copyright notice, terms of service or by other reasonable means, the name of such party or parties;

    I prefer using "\@ajlkn' instead of "HTML5 UP" for this attribution since the former sounds like a human user, whereas the later sounds like an orgainisation.

2. [*How to give attribution* - Creative Commons](https://creativecommons.org/use-remix/get-permission/)

### CODE REVIEW

**Comment by pacollins** [comment]
See comments on #47 on what we should do here. No where in the license does it demand that there be a direct link on each web page to his site. The work just has to give equal attribution, which it does everywhere else (if not more than enough). If he wants it on every page, it can be a comment at the top of the HTML. It does _not_ need to be visible, unless he can show where in the license it is required to be on every page.

File: i18n/en.toml [hunk 1]
```diff
@@ -46,3 +46,5 @@
   other = "Next Page"
 [previous_page]
   other = "Previous Page"
+[attribution]
+  other = "Powered by <a href='//gohugo.io' target='_blank' rel='noopener'>Hugo v{{ .Site.Hugo.Version }}</a>.  Theme <a href='//themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>Hugo Future Imperfect Slim</a> adapted from <a href='//html5up.net/future-imperfect' target='_blank' rel='noopener'>Future Imperfect</a>, which is licensed by <a href='//github.com/akjln' target='_blank' rel='noopener'>@akjln</a> under <a href='//creativecommons.org/licenses/by/3.0/' target='_blank' rel='noopener'>CC BY 3.0</a>."
```
Comment (pacollins) [inline i18n/en.toml]: All of my comments being said, if we decide to be extra nice and do it this way, it should be the following at most.
`Powered by <a href='//gohugo.io' target='_blank' rel='noopener'>Hugo v{{ .Site.Hugo.Version }}</a>.  Theme <a href='//themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>Hugo Future Imperfect Slim</a> adapted from <a href='//html5up.net/future-imperfect' target='_blank' rel='noopener'>Future Imperfect</a>, which is licensed under <a href='//creativecommons.org/licenses/by/3.0/' target='_blank' rel='noopener'>CC BY 3.0</a>.`

File: LICENSE.md [hunk 2]
```diff
@@ -25,6 +25,6 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 Original HTML5 UP's Future Imperfect theme
```
Comment (pacollins) [inline LICENSE.md]: Change the whole LICENSE to just be the CC 3.0 Text.

File: README.md [hunk 3]
```diff
@@ -203,24 +209,30 @@ highlightjsLang=["yaml", "groovy"]
...
 
-Please also refer to the [project contributors list](https://github.com/pacollins/hugo-future-imperfect-slim/graphs/contributors)
+Please also refer to the
+[project contributors list](https://github.com/pacollins/hugo-future-imperfect-slim/graphs/contributors)
 for everyone that has directly contributed to this version of the theme.
 
 ## License
 
-This theme is released under the MIT license. Please read the [license](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/LICENSE.md)
+The adaptation of the original theme is released under the MIT license.
```
Comment (pacollins) [inline README.md]: Our license should just change to CC 3.0.

File: README.md [hunk 4]
```diff
@@ -203,24 +209,30 @@ highlightjsLang=["yaml", "groovy"]
...
+"[Future Imperfect](https://html5up.net/future-imperfect)" is a theme by
+[@ajlkn](https://github.com/ajlkn) licensed under
```
Comment (pacollins) [inline README.md]: His organization HTML5 UP is the one who licensed it. Not him personally.

File: i18n/en.toml (hunk 1 above)
Comment (VincentTam) [reply i18n/en.toml]: Thanks for clarifications.  With due respect to \@balibebas, it's you who decide what's the best for our theme.  Despite my time and effort in searching these translations, having a concise footer is more important, as the last word in our theme name suggests: "Hugo, Future imperfect, Slim!"

Obviously, the "by \@ajlkn" part can be discarded.  I wonder if something like

> Powered by [Hugo v0.xx.x](//gohugo.io).  Theme [Hugo Future Imperfect]('//themes.gohugo.io/hugo-future-imperfect-slim/) adapted from [Future Imperfect](//html5up.net/future-imperfect).

would do.  Given that we've provided the URI to CC BY 3.0 and credits to the original author, conscious users will find out the details out the license themselves.  The "which is licensed under CC BY SA 3.0" in the footer has no role to play, for both parties

File: README.md (hunk 4 above)
Comment (VincentTam) [reply README.md]: Thanks for explaining.  My mistake at line 237.  The "SA" shouldn't be there.  Lemme change that.

**Comment by pacollins** [comment]
I don't love how the Copyright block looks right now. There are so many words because "Future Imperfect" and "Hugo Future Imperfect Slim" are so long. I like everything else.

Not to prolong the change or be a pain, but it is kinda an eyesore at the moment (and I realize the current version was my suggestion) that people will be wanting to remove anyway. If we want people to leave it alone, it has to look good.

**Comment by VincentTam** [comment]
> I don't love how the Copyright block looks right now. There are so many words because "Future Imperfect" and "Hugo Future Imperfect Slim" are so long. I like everything else. I'm not quite sure what the fix is.

I share your view about the visual effect.  That's why my demo site has the context root "fish-demo" instead of the name of this repo.  But I've no idea what the site footer should be.

**Comment by VincentTam** [comment]
My mind blocked me after a weekend's work on providing/improving [nested Staticman comments support](https://github.com/eduardoboucas/staticman/issues/35) for [Hugo Swift Theme](https://github.com/onweru/hugo-swift-theme/), [Introduction](https://github.com/victoriadrake/hugo-theme-introduction/), [Huginn](https://gitlab.com/VincentTam/huginn), and [my blog](https://lstu.fr/3s) using [Beautiful Hugo](https://themes.gohugo.io/theme/beautifulhugo/).

I've got the inspiration from [Hugo Swift Theme](https://github.com/onweru/hugo-swift-theme/) to suggest the following shortened footer:

> Designed by [@pacollins](https://themes.gohugo.io/hugo-future-imperfect-slim/), ported from [HTML5 UP](https://html5up.net/future-imperfect) [O 3.0](https://creativecommons.org/licenses/by/3.0/).

The link text and link target are the author and the URL for the work.  The former (author's name) is much the later (work's name).  The license and its URI is condensed to the icon 'O' illustrating the [Font Awesome icon for CC-BY](https://fontawesome.com/icons/creative-commons-by?style=brands), and a link is added to the URI of the human readable version of the license.  In the the whole footer, there's only four English words to be translated: "designed by" and "ported from".  I prefer keeping the current `{{ i18n "attribution" }}` since the word order (e.g. SVO, SOV) can be different in different languages.  (e.g. "I see you." SVO :gb: -> "Je te vois." :fr:)

**Comment by pacollins** [comment]
I like your solution. It has given me an idea:
```
<p class="copyright">
© 2019 Hugo Future Imperfect Slim <br> Theme: <a href="https://github.com/pacollins/hugo-future-imperfect-slim">Hugo Future Imperfect Slim</a><br>An <a href="https://html5up.net/future-imperfect">HTML5 UP port</a> | Powered by <a href="https://gohugo.io/">Hugo</a>
</p>
```
footer

- It provides the user a full line for copyright information.
- It creates a consistent three lines for all sizes.
- It provides Hugo advertising.
- It replaces my name with the theme name (because more than just me have contributed to the theme.

Thoughts?

**Comment by VincentTam** [comment]
> Thoughts? I'll be on for the next few hours, so no need to make a commit if you disagree or have other thoughts.

Looking at the visual results for English, the difference between my solution and yours is like amateur vs professional.  Wow!  What an instant pleasure! :grinning: :v:

However, thinking about this in :fr: French, I've discovered the possibility that the "consistent three lines" becomes four lines.

Screenshot_2019-08-14 Hugo Future Imperfect Slim in fr

If you're OK about that, then I'll proceed to writing other translations.

**Comment by VincentTam** [comment]
:fr: The French way to use the word *adapter* (to adapt) is *adapter qqch1 en qqch2* (to adapt into sth2 from sth1).  Some modern developers might be confortable with "adapté de qqch1 en qqch2", but some Francophones (like me) would rather prefer "une adaptation de qqch1 en qqch2"  (an adaptation from qqch1 to qqch2).

In the related discussion: https://forum.wordreference.com/threads/taken-and-adapted-from.2356960/ on Word Reference, Snarkhunter suggests using "_sth like "transposer (à partir) (de / du)..."._"

Applying his suggestion would give sth like

Screenshot_2019-08-14 Hugo Future Imperfect Slim in fr(1)

Btw, it's really hard to read the grave and acute accents in the footer.

**Comment by pacollins** [comment]
Does French not have a word similar to "port" that is shorter or would change the syntax?

**Comment by pacollins** [comment]
To clarify, I am fine with "Transpose" if you think it works and the lack of visible grave and accent is fine. [The French version of the Wikipedia article about porting uses "portage"](https://fr.wikipedia.org/wiki/Portage_informatique)

**Comment by VincentTam** [comment]
> Does French not have a word similar to "port" that is shorter or would change the syntax?

I don't know any shorter noun for that.  Probably a French won't (care to) know.  "Une adaptation" is the best way to say that.  I didn't see "transposé de" in this context before, but it's alright to use that.  Many Francophones are going to understand this.  Given that the user who has suggested this is from Ile-de-France, I'm OK with this.

My concern is that the "consistent three lines for all sizes" might be broken into four lines when translated into other languages.  After searching a French dictionary for that (probably non-existent word), we might run into this situation again in another language in the future.  This approach would be succumb to imperfection in the future.

:fr: In French, compound nouns are rare.  Perhaps "assurance habition" (housing assurance) would be the only possible one that I know.  (I've spent a few moment to search "le porte-parole": "la porte" (door) and "la parole" (speech), but the "porte" actually comes from the word "porter" (to wear, to uptake a role, etc))  In any case, if you search for a direct literal translation in French for "A HTML5 UP port", it would be in the form "un(e) xxx de HTML5 UP".

**Comment by pacollins** [comment]
I think the best course of action is to _encourage_ phrasing on to three lines, but not at the expense the translations integrity. That being said, go ahead and push the final commit. With your discretion, pick whichever phrasing translates best. If we have someone that comes along later with better phrasing, we can adjust it then.

> "A HTML5 UP port"

I am glad you said that. I wrote "An HTML5 UP Port," but that is incorrect. "A HTML5 UP Port"

**Comment by VincentTam** [comment]
Thanks for reminding me of "le portage".  That's more specific and precise.  Let's stick with this.

**Comment by pacollins** [comment]
For the license page, maybe we make it clear and model it after [this license](https://github.com/digitalcraftsman/hugo-strata-theme/blob/master/LICENSE.md), which is also an HTML5 UP Port.

**Comment by pacollins** [comment]
Going to hold for 30 minutes just to make sure you didn't have anything else. @VincentTam, if you read this message before then, feel free to squash and merge if you are ready.

**Comment by VincentTam** [comment]
The only issue is the Brazillian Portuguese translation.  I don't know Portuguese, so I can't help on that.

Screenshot_2019-08-15 Hugo Future Imperfect Slim em português

"Le portage" refers to the act of porting rather than the object to be ported.  In our case, our theme can represent the product as well as the process of porting, so I see no problem using "le portage" here, even though that's not a word-to-word translation.

**Comment by pacollins** [comment]
I am not sure who made the original Brazilian Portuguese, so I think it is fine. Hopefully, they will swing by and adjust it for their use.

**Comment by pacollins** [comment]
Based on my research, the proper is:

```
© 2019 Hugo Future Imperfect Slim
Tema: Hugo Future Imperfect Slim
Uma Porta HTML5 UP | Disponibilizado pelo Hugo
```
I am 100% sure on the "Powered by Hugo" the problem is that line is 3 characters two long. I think we can drop "Uma" without losing meaning and keep it on three lines. Worst case, someone comes in knowing Portuguese and corrects it for us. That would make it:

```
Tema: <a href='https://themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>Hugo Future Imperfect</a><br><a href='https://html5up.net/future-imperfect' target='_blank' rel='noopener'>Porta HTML5 UP</a> | Disponibilizado pelo <a href='https://gohugo.io/' title='{{ .Site.Hugo.Version }}' target='_blank' rel='noopener'>Hugo</a>
```

Also, I don't mind `title='{{ .Site.Hugo.Version }}`, but it is actually in the header:

https://github.com/pacollins/hugo-future-imperfect-slim/blob/d80bfe573cb7f423be4fe861a1dd108aac3eef0e/layouts/partials/head.html#L3

So I'd go change the Portuguese and then go ahead and merge.

**Comment by VincentTam** [comment]
> Based on my research, the proper is:
> 
> ```
> © 2019 Hugo Future Imperfect Slim
> Tema: Hugo Future Imperfect Slim
> Uma Porta HTML5 UP | Disponibilizado pelo Hugo
> ```
> 
> I am 100% sure on the "Powered by Hugo" the problem is that line is 3 characters two long. I think we can drop "Uma" without losing meaning and keep it on three lines. Worst case, someone comes in knowing Portuguese and corrects it for us. That would make it:
> 
> ```
> Tema: <a href='https://themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>Hugo Future Imperfect</a><br><a href='https://html5up.net/future-imperfect' target='_blank' rel='noopener'>Porta HTML5 UP</a> | Disponibilizado pelo <a href='https://gohugo.io/' title='{{ .Site.Hugo.Version }}' target='_blank' rel='noopener'>Hugo</a>
> ```

Dropping "uma" doesn't change the meaning, and "disponibilizado pelo Hugo" seems to be a better choice.  The only thing left is "porta".  From [Collins Portuguese Dictionary](https://www.collinsdictionary.com/dictionary/portuguese-english/porta), it seems to be the Portugese  word for "port" in computing, but "port" in computing has two meanings: the controller for connection with other machines; and the modified software from an existing one on other platform.  Using [Lexilogos](https://www.lexilogos.com/english/portuguese_dictionary.htm), I've found a few Portugest–English dictionaries, like [Michaelis](https://michaelis.uol.com.br/moderno-ingles/busca/portugues-ingles-moderno/porta/), [Word Reference](https://www.wordreference.com/pten/porta) and [Wiktionary](https://en.wiktionary.org/wiki/porta#Portuguese).  Their translations under "computing"  applies to the "port" as in "port number" or the gateway controlling incoming/outgoing data.  That's why I've put "Adaptado de".  Maybe a Spanish translation can shed us some light.

**Pull request merged**